shipyard/src/test/java/org/jclouds/shipyard/features/ServiceKeysApiLiveTest.java

```diff
@@ -53,13 +54,20 @@ public void testListServiceKeys() throws Exception {
       List<ServiceKey> possibleServiceKeys = api().listServiceKeys();
       assertNotNull(possibleServiceKeys, "possibleServiceKeys was not set");
       assertTrue(possibleServiceKeys.size() > 0, "Expected at least 1 ServiceKey but list was empty");
-      boolean serviceKeyFound = false;
-      for (ServiceKey possibleKey : possibleServiceKeys) {
-         if (possibleKey.key().equals(serviceKey)) {
-            serviceKeyFound = true;
+      ServiceKey possibleServiceKey = Iterables.find(possibleServiceKeys, new Predicate<ServiceKey>() {
+         @Override
+         public boolean apply(ServiceKey arg0) {
+            return (arg0.key().equals(serviceKey)) ? true : false;
          }
-      }
-      assertTrue(serviceKeyFound, "Expected but could not find ServiceKey amongst " + possibleServiceKeys.size() + " found");
```

Comment:
Maybe instead just use the overloaded version and supply the default "null" value? I think that would make things more readable IMO and is what I intended to do originally.